src/plugins/expression/expression-tile.test.tsx

```diff
@@ -18,6 +19,11 @@ jest.mock("../../hooks/use-stores", () => ({
   })
 }));
 
+// mock replaceSync so render happens
+jest.mock("mathlive", () => ({
```

Comment:
This allowed the test to run, but the tests are still doing nothing but asserting the element is there and has the default value.

Reply:
This is mocking out (i.e. eliminating) the `mathlive` library entirely, which makes these tests less-than-useful. It suggests that the problem may be related to jest's ability to load the `mathlive` library at all.